packages/lexical/src/nodes/LexicalParagraphNode.ts

```diff
@@ -48,8 +48,8 @@ export type SerializedParagraphNode = Spread<
 export class ParagraphNode extends ElementNode {
   ['constructor']!: KlassConstructor<typeof ParagraphNode>;
   /** @internal */
-  __textFormat: number;
-  __textStyle: string;
+  __textFormat?: number;
+  __textStyle?: string;
```

Comment:
This shouldn't change. There's no reason for these to be optional, they are always explicitly set in the constructor. There are a lot of other changes here that can be reverted.

```suggestion
  __textFormat: number;
  __textStyle: string;
```